jinahub/indexers/cache/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM jinaai/jina:1.0.2
+
+# install the third-party requirements
+RUN apt-get update && apt-get install --no-install-recommends -y gcc build-essential git
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+RUN pip install -r requirements.txt
+
+# for testing the image
+RUN pip install pytest && pytest -s -v tests
```

Comment:
I think we want to establish the `base` `from` syntax.

Reply:
done, sure